src/main/java/org/jabref/model/util/OptionalUtil.java

```diff
@@ -33,28 +32,17 @@ public static <T, U> boolean equals(Optional<T> left, Optional<U> right, BiPredi
         }
     }
 
-    /**
-     * No longer needed in Java 9 where {@code Optional<T>.stream()} is added.
-     */
-    public static <T> Stream<T> toStream(Optional<T> value) {
-        if (value.isPresent()) {
-            return Stream.of(value.get());
-        } else {
-            return Stream.empty();
-        }
-    }
-
     @SafeVarargs
     public static <T> List<T> toList(Optional<T>... values) {
-        return Stream.of(values).flatMap(optional -> toList(optional).stream()).collect(Collectors.toList());
+        return Stream.of(values).flatMap(Optional::stream).toList();
```

Comment:
We need to be careful here,  as we return an immutable list

Reply:
I check all calls. No one updates the result.

I also updated the JavaDoc accordingly.